### PR TITLE
Cleanup Alarm descriptions. Fixes #908

### LINKED ISSUE #908: GCS:SystemHealth: Audit to ensure all alarms are handled with descriptions
I suspect quite a few have been added without doing the necessary GCS work.

### PR DESCRIPTION
**Pull request by jpbarraca**

### CODE REVIEW

**Comment by tracernz** [comment]
```
RCC: Error in '/var/lib/jenkins/jobs/dron/workspace/Nodes/master/ground/gcs/src/plugins/systemhealth/systemhealth.qrc': Cannot find file 'html/TempBaro-Critical.html'
```

**Comment by mlyle** [comment]
Fixed #908

File: flight/Modules/Battery/battery.c [hunk 1]
```diff
@@ -185,6 +185,8 @@ static void batteryTask(void * parameters)
 				AlarmsClear(SYSTEMALARMS_ALARM_BATTERY);
 		} else {
 			flightBatteryData.Voltage = 0;
+			AlarmsSet(SYSTEMALARMS_ALARM_BATTERY, SYSTEMALARMS_ALARM_ERROR);
+			AlarmsSet(SYSTEMALARMS_ALARM_FLIGHTTIME, SYSTEMALARMS_ALARM_ERROR);
```
Comment (tracernz) [inline flight/Modules/Battery/battery.c:189]: Setting the alarm here means you are now required to have BOTH a current sensor and voltage sensor to use the battery module without error. It's also out of the scope of this PR. This should be reverted IMO.

File: flight/Modules/Battery/battery.c [hunk 2]
```diff
@@ -237,6 +239,8 @@ static void batteryTask(void * parameters)
 				AlarmsClear(SYSTEMALARMS_ALARM_FLIGHTTIME);
 		} else {
 			flightBatteryData.Current = 0;
+			AlarmsSet(SYSTEMALARMS_ALARM_BATTERY, SYSTEMALARMS_ALARM_ERROR);
+			AlarmsSet(SYSTEMALARMS_ALARM_FLIGHTTIME, SYSTEMALARMS_ALARM_ERROR);
```
Comment (tracernz) [inline flight/Modules/Battery/battery.c:243]: As above.

File: ground/gcs/src/plugins/systemhealth/html/Geofence-Warning.html [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    <meta content="">
+    <style></style>
+  </head>
+  <body style="color: black">
+    <h1>Geofence: Warning</h1>
+    <p>
+    The current position is beyond the warning radius.
+    </p>
+  </body>
+</html>
```
Comment (tracernz) [inline ground/gcs/src/plugins/systemhealth/html/Geofence-Warning.html:13]: Should be a linefeed here.

File: ground/gcs/src/plugins/systemhealth/html/GyroBias-Warning.html [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+    <meta content="">
+    <style></style>
+  </head>
+  <body style="color: black">
+    <h1>GyroBias: Warning</h1>
+    <p>
+    Gyroscope bias is too high, exceeding the warning threshold (10).
+    </p>
+  </body>
+</html>
```
Comment (tracernz) [inline ground/gcs/src/plugins/systemhealth/html/GyroBias-Warning.html:13]: Should be a linefeed here.

File: ground/gcs/src/plugins/systemhealth/html/I2C-Critical.html [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+    <meta content="">
+    <style></style>
+  </head>
+  <body style="color: black">
+    <h1>I2C: Critical</h1>
+    <p>
+    Could not initialize the I2C bus.
+    </p>
+  </body>
+</html>
```
Comment (tracernz) [inline ground/gcs/src/plugins/systemhealth/html/I2C-Critical.html:13]: Should be a linefeed here.

File: ground/gcs/src/plugins/systemhealth/html/Stabilization-Error.html [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+    <meta content="">
+    <style></style>
+  </head>
+  <body style="color: black">
+    <h1>Stabilization: Error</h1>
+    <p>
+    Internal error in the module.
+    </p>
+  </body>
+</html>
```
Comment (tracernz) [inline ground/gcs/src/plugins/systemhealth/html/Stabilization-Error.html:13]: Should be a linefeed here.

**Pull request merged**